docs/products/cassandra/concepts/cross-cluster-replication.rst

```diff
@@ -0,0 +1,132 @@
+
+With the replication strategy defined in the CREATE KEYSPACE query, every table in the keyspace is replicated within the cluster according to the specified strategy.
+
+``NetworkTopologyStrategy``
+'''''''''''''''''''''''''''
+
+The replication strategy that allows CCR in Aiven for Apache Cassandra is called `NetworkTopologyStrategy <https://docs.datastax.com/en/cassandra-oss/3.0/cassandra/architecture/archDataDistributeReplication.html#archDataDistributeReplication__nts>`__.
+
+``NetworkTopologyStrategy`` can be set up as the replication strategy when creating a keyspace on the cluster. The same CREATE KEYSPACE query can be used to specify the replication factor and a datacenter that data can be replicated to.
+
+Datacenter per service
+  For the CCR feature to work, the two services that cross-replicate need to be located in two different datacenters.
+
+Replication factor
+  The replication factor is defined in the CREATE KEYSPACE query to indicate the number of copies of the data to be saved per cluster (in each datacenter).
+
+.. seealso::
+
+    For more details on the replication factor for Apache Cassandra, see `NetworkTopologyStrategy <https://cassandra.apache.org/doc/4.1/cassandra/cql/ddl.html#networktopologystrategy>`__ in the Apache Cassandra documentation.
+
+CCR setup
+'''''''''
+
+To make CCR work on your services, you need a cluster comprising two Apache Cassandra services with CCR enabled. On the cluster, you need to issue the CREATE KEYSPACE request, specifying ``NetworkTopologyStrategy`` as a replication strategy along with desired replication factors.
+
+.. code-block:: bash
+   :caption: Example
+
+   CREATE KEYSPACE test WITH replication =  /
+   {                                        /
+    'class': 'NetworkTopologyStrategy',     /
+    'dc-1': 3,                              /
+    'dc-2': 3                               /
+   };
+
+Where ``dc-1`` and ``dc-2`` are the names of Apache Cassandra datacenter, which you can find in the Aiven console.
+
+CCR in action
+'''''''''''''
+
+With CCR enabled and configured, Apache Cassandra replicates each write in the keyspace to both services (datacenters) with an appropriate number of copies as per replication factor.
+
+Active-active model
+  Apache Cassandra uses an active-active model: clients have the choice of reading/writing either from one service or the other.
+
+Consistency level
+  The consistency level regulates how many nodes need to confirm they executed an operation for this operation to be considered successfully completed by the client. You can set up the consistency level to one of the allowed consistency level arguments depending on your needs.
+
+.. topic:: Examples
+
+    * LOCAL_QUORUM consistency level
+        The read is contained within the service you connect to (completes faster).
+    * QUORUM consistency level
+        The read produces more consistent results (replies from nodes of both services are required).
```

Comment:
I'd add a tradeoff, in the lines of "but fails if one of the regions is unavailable"